src/wasm2js.h

```diff
@@ -2048,13 +2050,13 @@ Ref Wasm2JSBuilder::processFunctionBody(Module* m,
 
 void Wasm2JSBuilder::addMemoryFuncs(Ref ast, Module* wasm) {
   Ref memorySizeFunc = ValueBuilder::makeFunction(WASM_MEMORY_SIZE);
-  memorySizeFunc[3]->push_back(ValueBuilder::makeReturn(
-    makeAsmCoercion(ValueBuilder::makeBinary(
-                      ValueBuilder::makeDot(ValueBuilder::makeName(BUFFER),
-                                            IString("byteLength")),
-                      DIV,
-                      ValueBuilder::makeInt(Memory::kPageSize)),
-                    AsmType::ASM_INT)));
+  memorySizeFunc[3]->push_back(ValueBuilder::makeReturn(makeAsmCoercion(
+    ValueBuilder::makeBinary(
+      ValueBuilder::makeDot(ValueBuilder::makeName(BUFFER),
+                            IString("byteLength")),
+      RSHIFT,
```

Comment:
`TRSHIFT`, as you suggest below, will make this `>>>`

Reply:
thanks!